src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -393,13 +396,13 @@ The **`stage`** denotes how this script (the `content`) should be inserted. Some
 
 **Why:** To retrieve the final config for use in other hooks.
 
-```js
+```ts
 'astro:config:done'?: (options: {
   config: AstroConfig;
   setAdapter: (adapter: AstroAdapter) => void;
   injectTypes: (injectedType: { filename: string; content: string }) => URL;
   logger: AstroIntegrationLogger;
-}) => void | Promise<void>;
+}: HookParameters<'astro:config:done'>) => void | Promise<void>;
```

Comment:
It should remove the previous type, now it is `varName: oneType: anotherType`

Same with all of them